src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -220,6 +222,48 @@ export class CategoryFacet extends Component implements IAutoLayoutAdjustableIns
      * This option can have negative effects on performance, and should only be activated when debugging.
      */
     debug: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    /**
+     * Specifies a JSON object describing a mapping of facet values to their desired captions. See
+     * [Normalizing Facet Value Captions](https://developers.coveo.com/x/jBsvAg).
+     *
+     * If this option is specified, the facet search box will be unavailable.
+     *
+     * **Examples:**
+     *
+     * You can set the option in the ['init']{@link init} call:
```

Comment:
The documentation here seems copy pasted from the standard facet, which makes sense. However, it's missing the important part of tailoring it to the special field format that this facet needs.

For example, if my facet value looks like this : 
```
foo
foo|bar
foo|bar|buzz
```
And I want to put a caption for `buzz`, the documentation would leave me to think I would need to put a caption for like so : 
```
"buzz": "A caption"
```
However, I understand it needs to be like this ? : 
```
"foo|bar|buzz" : "A caption" 
```

Reply:
The way it is right now, you would caption `buzz` by doing the former, i.e.

```
"buzz": "A caption"
```